pkg/cmd/server/api/types.go

```diff
@@ -968,6 +968,11 @@ type KubernetesMasterConfig struct {
 	// the server will not start. These values may override other settings in KubernetesMasterConfig which may cause invalid
 	// configurations.
 	ControllerArguments ExtendedArguments
+
+	// ReconcileMasterEndpoints indicates that OpenShift should update the list of endpoints for the
+	// "kubernetes" Service, removing any entry from the list whose corresponding master/apiserver is
+	// unhealthy or down.
+	ReconcileMasterEndpoints bool
```

Comment:
Do you have a related doc pull or something to explain why I'd ever want this to be false?  Seems like I'd want this always on or at least defaulted to true.